opba-banking-rest-impl/src/main/java/de/adorsys/opba/tppbankingapi/mapper/AccountListFacadeToRestMapper.java

```diff
@@ -0,0 +1,12 @@
+package de.adorsys.opba.tppbankingapi.mapper;
+
+import de.adorsys.opba.protocol.api.dto.result.body.AccountListBody;
+import de.adorsys.opba.tppbankingapi.Const;
+import de.adorsys.opba.tppbankingapi.ais.model.generated.AccountList;
+import org.mapstruct.Mapper;
+
+@Mapper(implementationPackage = Const.API_MAPPERS_PACKAGE)
```

Comment:
Use Spring component model for mappers and inject them accordingly. I.e.:
```
 @Mapper(componentModel = SPRING_KEYWORD, implementationPackage = XS2A_MAPPERS_PACKAGE)
```
Reference:
```
 @Mapper(componentModel = SPRING_KEYWORD, implementationPackage = XS2A_MAPPERS_PACKAGE)
    public interface FromRequest extends DtoMapper<ListTransactionsRequest, TransactionListXs2aContext> {
```

Reply:
done